CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
We require 3.4.3. I ran into an issue when I changed the requirement. I am nervous about making such a change (since it may change policy defaults) until we do the final branch for swift-3.

When that happens I am going to update the actual requirement.